packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -996,16 +1007,25 @@ module.exports = function (log, config, oauthdb) {
   Mailer.prototype.postVerifyEmail = function (message) {
     log.trace('mailer.postVerifyEmail', { email: message.email, uid: message.uid });
 
-    const templateName = 'postVerifyEmail';
-    const links = this._generateLinks(this.syncUrl, message.email, {}, templateName);
+    let templateName = 'postVerifyEmail';
+    let subject = gettext('Firefox Account verified');
+    const query = {};
+
+    if (message.style === 'trailhead') {
+      templateName = 'postVerifyTrailheadEmail';
+      subject = gettext('Welcome to the Firefox family');
```

Comment:
This string was changed to "Your Firefox Account is Confirmed" - see [the updated strings doc](https://docs.google.com/document/d/1jQwIsAMz7gFDUa0f22LVoAhKxo5vr2I3_xf8Kib2MVs/edit?ts=5ce20322#bookmark=id.991f3pix04kh).